Framework/Crystal/test/FilterPeaksTest.h

```diff
@@ -358,12 +390,14 @@ class FilterPeaksTest : public CxxTest::TestSuite {
     LeanElasticPeak peak(Mantid::Kernel::V3D(1, 1, 0), 1.0);
     peak.setIntensity(100.0);
     peak.setHKL(1, 1, 0);
+    peak.setIntHKL(V3D(1, 1, 0));
```

Comment:
Is it necessary to set `IntHKL` here - won't it default to the rounded `hkl` provided?

Reply:
I found that they are actually separate objects. `m_H, `m_K`, `m_L` are distinct from `m_intHKL`. Setting `HKL` by hand does not affect `intHKL` I found. This is normally not an issue when using algorithms to get the peaks.